R/project-configuration.R

```diff
@@ -222,25 +296,27 @@ ProjectConfiguration <- R6::R6Class(
       private$printLine("Relative path from working directory", getwd())
       private$printLine("Project Configuration File", fs::path_rel(as.character(self$projectConfigurationFilePath)))
       private$printLine("Model folder", fs::path_rel(as.character(self$modelFolder)))
-      private$printLine("Parameters folder", fs::path_rel(as.character(self$paramsFolder)))
-      private$printLine("Parameters file name", fs::path_rel(as.character(self$paramsFile)))
-      private$printLine("Individual parameters file name", fs::path_rel(as.character(self$individualsFile)))
-      private$printLine("Population parameters file name", fs::path_rel(as.character(self$populationParamsFile)))
-      private$printLine("Scenario definitions file name", fs::path_rel(as.character(self$scenarioDefinitionFile)))
-      private$printLine("Scenario applications definitions file name", fs::path_rel(as.character(self$scenarioApplicationsFile)))
-      private$printLine("Plot definitions file name", fs::path_rel(as.character(self$plotsFile)))
-      private$printLine("Experimental data folder", fs::path_rel(as.character(self$dataFolder)))
-      private$printLine("Experimental data file", fs::path_rel(as.character(self$dataFile)))
+      private$printLine("Configurations folder", fs::path_rel(as.character(self$configurationsFolder)))
+      private$printLine("Model Parameters", fs::path_rel(as.character(self$modelParamsFile)))
+      private$printLine("Individuals", fs::path_rel(as.character(self$individualsFile)))
+      private$printLine("Populations", fs::path_rel(as.character(self$populationsFile)))
+      private$printLine("Populations Folder", fs::path_rel(as.character(self$populationsFolder)))
+      private$printLine("Scenarios", fs::path_rel(as.character(self$scenariosFile)))
+      private$printLine("Applications", fs::path_rel(as.character(self$applicationsFile)))
+      private$printLine("Plots", fs::path_rel(as.character(self$plotsFile)))
+      private$printLine("Data folder", fs::path_rel(as.character(self$dataFolder)))
+      private$printLine("Data file", fs::path_rel(as.character(self$dataFile)))
       private$printLine("Data importer configuration", fs::path_rel(as.character(self$dataImporterConfigurationFile)))
-      private$printLine("Compound Properties File", fs::path_rel(as.character(self$compoundPropertiesFile)))
       private$printLine("Output folder", fs::path_rel(as.character(self$outputFolder)))
       invisible(self)
     },
     #' @description Export ProjectConfiguration object to ProjectConfiguration.xlsx
+    #' @param path A string representing the path to the exported file.
```

Comment:
Add that the path is relative to the current wd.

Reply:
If an absolute path is provided, then it will be written at absolute location. Otherwise it's relative. As this is the case for all path arguments in base R this is to be expected, however, I will add a mention to this in the docstring.

Reply:
Yeah maybe I just didn't use R that much recently :D but it took me a while to understand where the new file is..